payment-platform-adapter/src/interfaces/iFRMTransaction.ts

```diff
@@ -0,0 +1,216 @@
+import { iMLTransaction } from './iMLTransaction';
+
+class PaymentIdentification {
```

Comment:
Is there any specific reason to use `class` instead of an interface? I think these are will be used as types of ISO data.

Reply:
Also, the code looks nice, it's not a suggestion just question :)

Reply:
I instantiate the master object in the constructor, by converting the Mojaloop Quote message into the ISO formatted message, which can't be done with an interface - I'd have to make a separate method to do that. Now it's all in one file. Hence the class. All the sub objects is already initialized as well by calling their base constructor, so it just made it a lot simpler to build the master constructor.